src/main/java/com/bhoruka/bloodbank/model/response/SearchCampResponse.java

```diff
@@ -0,0 +1,26 @@
+package com.bhoruka.bloodbank.model.response;
+
+import java.util.Date;
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public class SearchCampResponse {
```

Comment:
When you search for something with some criteria, there can be multiple matches. This class cannot handle that.
Please return a List of results instead.